src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -282,7 +289,7 @@ internal void AddProject(AbstractProject project)
             {
                 StartPushingToWorkspaceAndNotifyOfOpenDocuments(SpecializedCollections.SingletonEnumerable(project));
             }
-            else
+            else if (_batchingProjectLoads)
```

Comment:
This seems like there is another bug there. Now we're just dropping it on the floor, and it will never become part of the workspace.